web3-functions/redstone/package.json

```
@@ -1,16 +1,16 @@
 {
-  "name": "lisk-contracts",
+  "name": "web3-functions",
```

Since the package.json is now moved inside of the redstone sub-directory.
```suggestion
  "name": "web3-functions-redstone",
```